calligraphy/src/main/java/uk/co/chrisjenx/calligraphy/CalligraphyConfig.java

```diff
@@ -131,6 +138,15 @@ public int getAttrId() {
         return mAttrId;
     }
 
+    /**
+     *
+     * @param viewClass class to be ignored by Calligraphy
+     * @return true if class should not be injected, false otherwise
+     */
+    public boolean isIgnoredClass(Class<?> viewClass) {
```

Comment:
That should be in the Config. Just iterating across the list should not be done inside of the Config. Config should be just immutable state storage.